active/teachingtip/TeachingTipSpec.md

```diff
@@ -0,0 +1,571 @@
+# Teaching tip
+
+## Description
+
+A teaching tip is a content-rich nonmodal flyout that provides contextual information. A teaching tip looks like a flyout, except that it may target a UI element with a tail or be persistent and require the user to manually dismiss it. A teaching tip is often used for informing, reminding, and teaching users about important and new features that may enhance their experience. 
+
+**Important APIs:** [TeachingTip class]() [Link TODO]
+
+A teaching tip may also be light-dismiss enabled or used without a tail indicating reference to any specific UI element. 
+
+
+### Is this the right control? 
+
+Use a **TeachingTip** control to inform users about new and important updates and features, focus users on a component to advise them about its options or state, remind users of nonessential options that would improve their experience, or teach users how a task should be completed. 
+
+Because a teaching tip is impermanent, it would not be the recommended control for notifying users about important errors and statuses, prompting users for event handling, nor displaying a list of items or events that have been handled. 
+
+<!-- Explain why TT is not recommended for notifications --> 
+
+
+## Examples
+
+The teaching tip has several states, including these notable ones.
+
+A teaching tip can point to a specific UI element to enhance contextual clarity of the information it is presenting. 
+
+<a href="url"><img src="TeachingTipSampleApp.jpg" height="600"></a>
```

Comment:
I talked with the content writer and ![alt text](image path) is what we should be using to make sure the content is copy/paste-able. I need to check what folder name we should use.

Reply:
I believe we lost the inline thread when I fixed a separate issue on the same line. I am pinging here again for visibility, @jesbis and @chigy.

Reply:
You can see all the docs [here](https://github.com/MicrosoftDocs/windows-uwp/blob/docs/windows-apps-src/design/controls-and-patterns) - e.g. passwordbox [page](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/password-box) & [source](https://raw.githubusercontent.com/MicrosoftDocs/windows-uwp/docs/windows-apps-src/design/controls-and-patterns/password-box.md).

It looks like the docs do occasionally use image tags and other HTML (e.g. a few on the [buttons page](https://raw.githubusercontent.com/MicrosoftDocs/windows-uwp/docs/windows-apps-src/design/controls-and-patterns/buttons.md)), but for the most part it's just markdown on pre-sized images.

FWIW pre-sizing images is also preferable to setting sizes in the doc because:
- it'll result in smaller filesizes, which is better for a) reducing repo size and b) reducing page load times
- image editors will often do better-quality scaling than browsers
- scaling images with text looks particularly bad - it's better to just rasterize at the right size when saving the image

Reply:
To reduce the amount of work to convert this spec into a publishable format, please use markdown wherever possible.  Within our content, we generally don't use images as link anchors. You can improve the accessibility of this approach by adding alt text to the image, but I recommend using an image + markdown text. 

Re: image sizing. When this is displayed on docs.microsoft.com, the system resizes the image to fit the current width of the content column.

Reply:
Should we provision this repo with docs.microsoft.com so folks can get a preview of what the spec looks like? You'd view the specs by going to review.docs.microsoft.com/windows/Microsoft-ui-XAML-specs/

Reply:
@mijacobs - solved the issue where I wasn't able to comment back so bringing this back. Let's figure out the flow. Sounds nice, that way, we can verify the output as we write rather than find out later that it doesn't work, but I want to see how it fits in the overall process and if that becomes confusing.

Reply:
@chigy In order to set up publishing, I'll need admin access to this repo.

Reply:
@jesbis - If you agree with the authoring flow, please give access to Mike. If not, let's discuss.

Reply:
I have updated all images to use markdown and included alt text.